safedelete/locale/de/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,46 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: django-safedelete\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2014-10-28 18:03+0100\n"
+"PO-Revision-Date: 2018-03-07 12:10+0100\n"
+"Last-Translator: Ronny Vedrilla\n"
+"Language-Team: django-safedelete <LL@li.org>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"X-Generator: Poedit 2.0.6\n"
+"Language: fr\n"
+
+#: admin.py:31
+msgid "Name"
+msgstr "Name"
+
+#: admin.py:109 admin.py:116
+#, python-format
+msgid "Successfully undeleted %(count)d %(items)s."
+msgstr "%(count)d %(items)s gelöscht."
```

Comment:
The English one says "undeleted" while the German one means "deleted".

Reply:
Oops.